store/rootmulti/store.go

```diff
@@ -813,7 +820,15 @@ func (rs *Store) loadCommitStoreFromParams(key types.StoreKey, id types.CommitID
 		panic("recursive MultiStores not yet supported")
 
 	case types.StoreTypeIAVL:
-		store, err := iavl.LoadStore(db, id, rs.lazyLoading)
+		var store types.CommitKVStore
+		var err error
+
+		if params.initialVersion == 0 {
+			store, err = iavl.LoadStore(db, id, rs.lazyLoading)
+		} else {
+			store, err = iavl.LoadStoreWithInitialVersion(db, id, rs.lazyLoading, params.initialVersion)
+		}
```

Comment:
This check isn't strictly necessary, since `LoadStore()` is equivalent to `LoadStoreWithInitialVersion(..., 0)`. But might be good to be explicit about it anyway.